CMakeLists.txt

```diff
@@ -42,6 +42,19 @@ list(APPEND CMAKE_MODULE_PATH ${AWS_MODULE_PATH})
 # Local CMake scripts
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
+if (ANDROID)
+    set(ANDROID_DEPS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/target/cmake-build/deps/${ANDROID_ABI})
+    set(LibCrypto_INCLUDE_DIR "${ANDROID_DEPS_DIR}/libcrypto/include" CACHE INTERNAL "")
+    set(LibCrypto_STATIC_LIBRARY "${ANDROID_DEPS_DIR}/libcrypto/lib/libcrypto.a" CACHE INTERNAL "")
+    set(LibCrypto_SHARED_LIBRARY "${ANDROID_DEPS_DIR}/libcrypto/lib/libcrypto.so" CACHE INTERNAL "")
+
+    if (NOT EXISTS ${LibCrypto_INCLUDE_DIR}/openssl/crypto.h OR
+        NOT EXISTS ${LibCrypto_STATIC_LIBRARY} OR
+        NOT EXISTS ${LibCrypto_SHARED_LIBRARY})
```

Comment:
Because right now every binary dist and build we have has them in there. it's mostly a completeness thing. I could remove this.

Reply:
just seems the dynamic build isn't that useful, especially on unix systems where we use it since you'd either have to rely on rpath or LD_LIBRARY_PATH for it to load.